### PR TITLE
Issue 78

### PR DESCRIPTION
**Pull request by rgrcoding**
## Checklist
- [ ] Targeted PR against correct branch
- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For Admin Use:
- [ ] Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- [ ] Reviewers Assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]"

### CODE REVIEW

**Comment by rgrcoding** [comment]
@marcotradenet @alessio @gsora Take a look if this is what was expected. :D

File: x/id/types/priv_key_test.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetPemPublicKeyFromPemPrivKeyFile(t *testing.T) {
+	privKey, err := LoadRSAPrivKeyFromDisk("testfiles/priv_key.pem")
```
Comment (alessio) [inline x/id/types/priv_key_test.go:11]: The dir should be `testdata`

Comment (alessio) [reply x/id/types/priv_key_test.go:11]: As per Golang standard conventions

Comment (rgrcoding) [reply x/id/types/priv_key_test.go:11]: thanks

File: x/id/types/priv_key_test.go [hunk 2]
```diff
@@ -2,18 +2,19 @@ package types
 
 import (
 	"io/ioutil"
+	"path"
```
Comment (alessio) [inline x/id/types/priv_key_test.go:5]:
```suggestion
	"path/filepath"
```

File: x/id/types/priv_key_test.go [hunk 3]
```diff
@@ -2,18 +2,19 @@ package types
...
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
 func TestGetPemPublicKeyFromPemPrivKeyFile(t *testing.T) {
-	privKey, err := LoadRSAPrivKeyFromDisk("testfiles/priv_key.pem")
+	privKey, err := LoadRSAPrivKeyFromDisk(path.Join("testdata", "priv_key.pem"))
```
Comment (alessio) [inline x/id/types/priv_key_test.go:12]:
```suggestion
	privKey, err := LoadRSAPrivKeyFromDisk(filepath.Join("testdata", "priv_key.pem"))
```

File: x/id/types/priv_key_test.go [hunk 4]
```diff
@@ -2,18 +2,19 @@ package types
...
 	require.NoError(t, err)
 
 	pemPubKey := PublicKeyToPemString(&privKey.PublicKey)
 
-	expectePemPubKey, err := ioutil.ReadFile("testfiles/pub_key.pem")
+	expectePemPubKey, err := ioutil.ReadFile(path.Join("testdata", "pub_key.pem"))
```
Comment (alessio) [inline x/id/types/priv_key_test.go:17]:
```suggestion
	expectePemPubKey, err := ioutil.ReadFile(filepath.Join("testdata", "pub_key.pem"))
```

**Comment by rgrcoding** [comment]
We can merge this one?

**Pull request merged**